WebDriverAgentLib/Categories/XCUIElement+FBUtilities.m

```diff
@@ -133,18 +133,49 @@ - (void)fb_requestSnapshot:(XCAccessibilityElement *)accessibilityElement reply:
   });
   id<XCTestManager_ManagerInterface> proxy = [FBXCTestDaemonsProxy testRunnerProxy];
   if (useNewSnapshotAPI) {
-    [proxy _XCT_requestSnapshotForElement:self.lastSnapshot.accessibilityElement
+    [proxy _XCT_requestSnapshotForElement:self.fb_accessibilityElementBySnapshot
                                attributes:axAttributes
                                parameters:defaultParameters
                                     reply:block];
   } else {
-    [proxy _XCT_snapshotForElement:self.lastSnapshot.accessibilityElement
+    [proxy _XCT_snapshotForElement:self.fb_accessibilityElementBySnapshot
                         attributes:axAttributes
                         parameters:defaultParameters
                              reply:block];
   }
 }
 
+/**
+  Whether 'includingNonModalElements' is available
+
+  @param query The XCUIElementQuery to check if it has 'includingNonModalElements'
```

Comment:
I would say the names of these methods are obvious, so detailed docstrings are not really needed. We could just add some comments here and there, where it is not rally clear on what is being done and why